services/brig/src/Brig/API/User.hs

```diff
@@ -525,6 +544,8 @@ changeEmail u email = do
     -- The user already has an email address and the new one is exactly the same
     Just current | current == em -> return ChangeEmailIdempotent
     _ -> do
+      when (userManagedBy usr == ManagedByScim && allowScim == ForbidSCIMUpdates) $
```

Comment:
like above, you *could* change this to `unless ...`.

Reply:
done